### PR TITLE
Documentation: assign-collaboration: Fix typo

### PR DESCRIPTION
**Pull request by hanyujie2002**

### CODE REVIEW

File: Documentation/teaching/so2/assign-collaboration.rst [hunk 1]
```diff
@@ -121,7 +121,7 @@ Follow the seven rules of writing a good commit message: https://cbea.ms/git-com
 Add `TODOs` with each team member tasks. Try to split the work evenly.
```
Comment (dbaluta) [inline Documentation/teaching/so2/assign-collaboration.rst:122]: @hanyujie2002  for next patches can you please also your Signed-off-by line?

Comment (hanyujie2002) [reply Documentation/teaching/so2/assign-collaboration.rst:122]: > @hanyujie2002 for next patches can you please also your Signed-off-by line?

Sorry, the Linux repo is too huge, which causes me finding myself reluctant to clone this repo on my personal computer. So I make commits directly on GitHub, which makes signing-off impossible.

Comment (dbaluta) [reply Documentation/teaching/so2/assign-collaboration.rst:122]: So, when you create commit directly on github isn;t possible to add by hand:

Signed-off-by: First Name Last Name <email> 

in your commit message?

This sounds strange. 

How do you add the commit subject then?

Comment (hanyujie2002) [reply Documentation/teaching/so2/assign-collaboration.rst:122]: On, I see. Thanks for your instruction. 👍

**Pull request merged**